src/util/makeEnterClickable.ts

```diff
@@ -2,6 +2,12 @@ import { forceUpdate } from '@stencil/core';
 
 const eventHandlers: WeakMap<HTMLElement, CallBacks> = new WeakMap();
 
+/**
+ * Overrides the default browser behavior for clickable elements
+ * When focused and pressing down enter, avoids calling onClick repeatedly
+ * @param {HTMLElement} element the clickable element
+ */
+// eslint-disable-next-line sonarjs/cognitive-complexity
```

Comment:
We should not disable this rule :smile:

Reply:
Hang on, what happened here?

<img width="1064" alt="image" src="https://github.com/Lundalogik/lime-elements/assets/154725/837758d3-83ce-4fe1-9235-ee10e3b9bfef">

Was this thread resolved by mistake, or was there a good reason to resolve it with no change? @LawrenceBorst